docs/src/modules/components/HighlightedCodeWithTabs.tsx

```diff
@@ -4,24 +4,40 @@ import { Tabs, TabsOwnProps } from '@mui/base/Tabs';
 import { TabsList as TabsListBase } from '@mui/base/TabsList';
 import { TabPanel as TabPanelBase } from '@mui/base/TabPanel';
 import { Tab as TabBase } from '@mui/base/Tab';
-import HighlightedCode from './HighlightedCode';
+import HighlightedCode from 'docs/src/modules/components/HighlightedCode';
 
-const TabList = styled(TabsListBase)(({ theme }) => ({
-  padding: 6,
+export const TabList = styled(TabsListBase)<{
```

Comment:
Seems simpler to work on a more global namespace

```suggestion
export const CodeTabList = styled(TabsListBase)<{
```

x the other one